src/constants.js

```diff
@@ -13,27 +13,27 @@ const BROWSER_ALIASES_MAP = {
   Generic: 'generic',
   Googlebot: 'googlebot',
   'Internet Explorer': 'ie',
-  'K-Meleon': 'kMeleon',
+  'K-Meleon': 'k_meleon',
   Maxthon: 'maxthon',
   'Microsoft Edge': 'edge',
   'MZ Browser': 'mz',
   'NAVER Whale Browser': 'naver',
   Opera: 'opera',
-  'Opera Coast': 'operaCoast',
-  PhantomJS: 'phantomJS',
+  'Opera Coast': 'opera_coast',
+  PhantomJS: 'phantom_js',
```

Comment:
I thought more about using underscore only for spaces and special characters, but camel-cased letters might be simply ignored. Like this: `opera_coast`, but `seamonkey`. Because, it again confusing in cases like `PhantomJS` → `phantom_js`, instead of `phantom_j_s`. I'd offer here simple lowercase for everything, and underscore only for special chars. `phantomjs`, `opera_coast`, `k_meleon`, `blackberry`. What do you think of this?